iroha/iroha.py

```diff
@@ -57,18 +66,24 @@ def hash(proto_with_payload):
         return hash
 
     @staticmethod
-    def _signature(message, private_key):
+    def _signature(message, private_key, use_multihash=False, multihash_code=0x12):
```

Comment:
```suggestion
    def _signature(message, private_key, use_multihash=False, multihash_code=0xed):
```

Reply:
fixed in my [patch](https://github.com/hyperledger/iroha-python/pull/54#issuecomment-642658788)